src/components/Checkbox/Checkbox.module.css

```diff
@@ -7,158 +7,109 @@
 /**
  * A custom individual checkbox control
  * 1) Outer wrapper div for custom control
- * 2) Determines width and height of the control
+ * 2) Grid display aligns label and the checkbox
  * 3) Prevents control from being "squished" in 
  *    a flexbox parent
  */
-.checkbox {
+.checkbox { /* 1 */
   position: relative;
-  display: block;
-  width: var(--eds-size-2-and-half); /* 2 */
-  height: var(--eds-size-2-and-half); /* 2 */
-  flex: none; /* 3 */
+
+  display: grid; /* 2 */
+  align-items: flex-start;
+  grid-template-columns: min-content 1fr;
+  gap: var(--eds-size-1);
+}
+
+/**
+ * Text labelling the checkbox.
+ */
+.label {
+  color: var(--eds-theme-color-form-label-text);
+
+  /* Shift label so the checkbox always appears vertically centered
+  with the *first* line of text, in case label wraps. */
+  /* margin-top: calc((var(--checkbox-svg-size) - var(--label-line-height)) / 2); */
```

Comment:
Label text used to be centered via a margin-top like so, but I changed centering by setting line-height (see `.label--sm` and `.label--md` styling). Lmk if should revert to this calculated margin-top styling, otherwise will remove these comments.

Reply:
oh nice! idk why I didn't think of that 🙂  maybe we can set the line height in this `.label` class to DRY? and let's make sure to call out in the comment that the label is vertically centered with the first line of text (not the entire block of text)

Reply:
I tried setting line-height in `.label` for aridness but if the `.label` block is placed higher than `.label--sm` or `.label--md` blocks, the line-heights being set in those variants override the line-height in `.label`. Didn't think having styling dependent on order in stylesheet was good, but lmk if we should proceed with that since I can place the `.label` lower.

Reply:
ohhh right of course, that makes sense! thanks for explaining 👍